amy/urls.py

```diff
@@ -44,6 +44,8 @@
 
     # Login with GitHub credentials
     url('', include('social.apps.django_app.urls', namespace='social')),
+
+    url(r'^forms/', include('extforms.urls')),
```

Comment:
Could you move this before GitHub login, so that we go from more specific URL patterns (`'^forms/'`) to more general ones (`''`).